GliaWidgetsTests/Mocks/UIViewControllerCoordinator.Mock.swift

```diff
@@ -0,0 +1,20 @@
+@testable import GliaWidgets
+import UIKit
+
+extension UIViewControllerCoordinator {
+    static let mock: UIViewControllerCoordinator = Mock()
```

Comment:
This will create one instance and will keep it in memory thus providing same instance.
 I think it should be a method that will produce new instance every time.

Reply:
That is very good point that I missed